matplotlib2tikz/legend.py

```diff
@@ -181,6 +187,17 @@ def draw_legend(data, obj):
             alignment = None
             break
 
+    # Set color of lines in legend
+    data['legend colors'] = []
+    for handle in obj.legendHandles:
+        try:
+            data, legend_color, _ = mycol.mpl_color2xcolor(data,
+                                                           handle.get_color())
+            data['legend colors'].append('\\addlegendimage{no markers, %s}\n'
+                                         % legend_color)
+        except:
```

Comment:
Awkward, checked it again with polar plots without try statement and it works. Apparently, I can't
remember why I used try except...
So I guess we can skip it, too :D